README.md

```diff
@@ -54,85 +72,70 @@ The Optimism Immunefi program offers up to $2,000,042 for in-scope critical vuln
+We always start with `rc.1` rather than `rc`.
+
+Tags of the form `v<semver>`, such as `v1.1.4`, indicate releases of all Go code only, and **DO NOT** include smart contracts.
+This naming scheme is required by Golang.
+In the above list, this means these `v<semver` releases contain all `op-*` components, and exclude all `contracts-*` components.
 
-This repository generally follows [this Git branching model](https://nvie.com/posts/a-successful-git-branching-model/).
-Please read the linked post if you're planning to make frequent PRs into this repository.
+`op-geth` embeds upsteam geth’s version inside it’s own version as follows: `vMAJOR.GETH_MAJOR GETH_MINOR GETH_PATCH.PATCH`.
+Basically, geth’s version is our minor version.
+For example if geth is at `v1.12.0`, the corresponding op-geth version would be `v1.101200.0`.
+*Note that we pad out to three characters for the geth minor version and two characters for the geth patch version.
+Since we cannot left-pad with zeroes, the geth major version is not padded.
 
-### Production branch
+The full set of components that have releases are:
 
-The production branch is `master`.
-The `master` branch contains the code for latest "stable" releases.
-Updates from `master` **always** come from the `develop` branch.
+- `op-node`
+- `op-batcher`
+- `op-proposer`
+- `op-challenger`
+- `contracts-L1`
+- `contracts-L2`
+
+All other components and packages should be considered development components only and do not have releases.
 
 ### Development branch
 
 The primary development branch is [`develop`](https://github.com/ethereum-optimism/optimism/tree/develop/).
 `develop` contains the most up-to-date software that remains backwards compatible with the latest experimental [network deployments](https://community.optimism.io/docs/useful-tools/networks/).
 If you're making a backwards compatible change, please direct your pull request towards `develop`.
 
-**Changes to contracts within `packages/contracts-bedrock/src` are usually NOT considered backwards compatible and SHOULD be made against a release candidate branch**.
-Some exceptions to this rule exist for cases in which we absolutely must deploy some new contract after a release candidate branch has already been fully deployed.
-If you're changing or adding a contract and you're unsure about which branch to make a PR into, default to using the latest release candidate branch.
+**Changes to contracts within `packages/contracts-bedrock/src` are usually NOT considered backwards compatible and SHOULD be made against a feature branch**.
+Some exceptions to this rule exist for cases in which we absolutely must deploy some new contract after a tag has already been fully deployed.
+If you're changing or adding a contract and you're unsure about which branch to make a PR into, default to using a feature branch.
 See below for info about release candidate branches.
 
-### Release candidate branches
```

Comment:
AFAICT we no longer use changeset or release candidate branches. The way to trigger releases was outdated and doesn't need to be in this README anyway so I removed that too

Reply:
Yep, both of these are true